Addon.lua

```diff
@@ -469,9 +469,9 @@ end
 function Addon:UPDATE_OVERRIDE_ACTIONBAR()
 	local inCamera = false
 	for i = 1, 40 do
-		local name = UnitBuff("player", i)
-		if not name or cameraBuffs[name] then
-			inCamera = name and cameraBuffs[name]
+		local playerAura = C_UnitAuras.GetBuffDataByIndex("player", i)
```

Comment:
Not sure how you'd prefer to do stuff like this / handle the fact that the returned object is a little more complex; as mentioned, newish to Lua so the truthiness of everything isn't as immediately clear - this can be done more elegantly but lol